benchmark/child_process/child-process-exec-stdout.js

```diff
@@ -29,7 +30,12 @@ function main(conf) {
   });
 
   setTimeout(function() {
-    child.kill();
     bench.end(bytes);
+    if (process.platform === 'win32') {
+      // Sometimes there's a yes.exe process left hanging around on Windows...
+      child_process.execSync(`taskkill /f /t /pid ${child.pid}`);
```

Comment:
Is this the best way to do it, it seems so cruel 😭
Now seriously, `child.kill` should work, aren't we masking a real problem?